DevTools/UserspaceEmulator/SoftCPU.cpp

```diff
@@ -1672,11 +1672,17 @@ void SoftCPU::FISUB_RM32(const X86::Instruction& insn)
     ASSERT(!insn.modrm().is_register());
     auto m32int = (i32)insn.modrm().read32(*this, insn).value();
     // FIXME: Respect shadow values
-    auto f64 = (long double)m32int;
```

Comment:
(removing the var here because a long double doesn't have 64 bits)

Reply:
Oh right!